hyperloglog_test.go

```diff
@@ -576,6 +576,58 @@ func TestHLLTC_Marshal_Unmarshal_Count(t *testing.T) {
 	}
 }
 
+func TestHLLTC_Clone_Basic(t *testing.T) {
+	sk1 := NewTestSketch(4)
+	sk2 := sk1.Clone()
+
+	testSketchProperties(sk1, sk2, t)
+}
+
+func TestHLLTC_Clone_Populated(t *testing.T) {
+	sk1 := NewTestSketch(16)
+
+	sk1.Insert(toByte(0x00010fffffffffff))
+	sk1.Insert(toByte(0x0002ffffffffffff))
+	sk1.Insert(toByte(0x0003000000000000))
+	sk1.Insert(toByte(0x0003000000000001))
+	sk1.Insert(toByte(0xff03700000000000))
+
+	n := sk1.Estimate()
+	if n != 5 {
+		t.Error(n)
+	}
+
+	sk2 := sk1.Clone()
+
+	testSketchProperties(sk1, sk2, t)
+}
+
+func testSketchProperties(sk1, sk2 *Sketch, t *testing.T) {
```

Comment:
dont pass ``` t *testing.T```
create a ```func isSketchEqual(sk1, sk2 *Sketch)``` bool
the error should be raised by the calling function with 

```
if !isSketchEqual(sk1, sk2){
		t.Error("Expected %v, got %v1", sk1, sk2)
}
```